__tests__/body-customisation.test.ts

```diff
@@ -0,0 +1,25 @@
+import { setBodyParameters} from '../src/automation'
+import { BodyParam} from '../src/types/index'
+import { sampleMessages } from '../src/utils/setupTests'
+import typeorm = require('typeorm')
+
+describe('Unit Tests', () => {
+
+  test('Test setBodyParameters with 2 parameters set', async () => {
+    const body = "<h1>PARAMS_Lastname PARAMS_Firstname Send automated messages with Candymail</h1><p>Now with HTML support</p><a href='https://saasbase.dev/candymail'>Learn more here</a>"
+    const params=[{key:"Firstname",value:"Harry"},{key:"Lastname",value:"GBAGUIDI"}]
+    let response = setBodyParameters(body, params)
+    let counter=0
+    params?.forEach(element => {
+      counter=counter+(response.match(/PARAMS_/g) || []).length
+    });
+    expect(counter).toBe(0)
```

Comment:
We could prob shorten this to: `counter = (response.match(/PARAMS_/g) || []).length;` to see if there is any PARAMS_ that wasn't replaced.